protocols/imap/src/test/java/org/apache/james/imap/processor/GetAnnotationProcessorTest.java

```diff
@@ -90,9 +93,10 @@ private void initAndMockData() {
         mailboxSession = new MockMailboxSession("username");
         inbox = MailboxPath.inbox(mailboxSession);
         KEYS = ImmutableSet.of("anyKey");
-        annotationRequest = new GetAnnotationRequest(TAG, ImapCommand.anyStateCommand("Name"), ImapConstants.INBOX_NAME);
+        annotationRequestBuilder = new GetAnnotationRequest.GetAnnotationRequestBuilder(TAG, ImapCommand.anyStateCommand("Name")).mailboxName(ImapConstants.INBOX_NAME);
```

Comment:
For builder, we use to intruce a method : 

```
 public static Builder builder() { return new Builder() }
```

Directly in GetAnnotationRequest  ^^

TAG should be set by a method of the builder. Idem for name. So you better have no parameters on builder instanciation.